src/components/About/style.scss

```diff
@@ -7,7 +7,7 @@
 .square-ornament {
 	height: 200px;
 	width: 200px;
-	border-radius: 20px;
+	border-radius: 40px;
```

Comment:
For all of these updates I've assumed the ratio of 20% border-radius w/ respect to width (40/200)

I'm not sure this is correct, however, b/c I've seen other ratios used in #71 such as 15/50 and 40/150. Defer to @chern @vohndernet @austinkeil as they know the most about the rebranding style guide.

We should const-ify this somehow as the `border-radius` css property takes percentages as a value. Like so:
```
App.scss:
$diamond-border-radius-percentage: X%;
---
Other.scss
.square {
    width: Ypx;
    border-radius: $$diamond-border-radius-percentage;
}
```

Reply:
This is a really good point, let's get this PR through and @vohndernet and I will work on putting a standard in place and raise the change in another PR.